apps/web/src/composables/users.ts

```diff
@@ -80,6 +92,8 @@ export default function useUsers () {
             }
             const response = await fetch(`${usersBaseURL}/users/add-sub-account`, requestOptions)
             const json = await response.json()
```

Comment:
const { user } = await response.json()

Reply:
Can't call it `user` due to naming collision, so called it `userData` instead.